### DISCUSSION TITLE
Change default timeout for `findBy*` and `waitFor` functions?

### DISCUSSION THREAD

**Discussion by jhpedemonte** [storybookjs/storybook#29610] [Help] [answered]
### Summary

The functions provided by testing-library (such as `findBy*` and `waitFor`) have a default timeout of 1000ms. Is there a way to change this default across all these functions, instead of having to add the option to each function call? This would be helpful in CI environments where the tests run slower.

**Comment by IanVS** [comment, marked as answer]
This can be done using:

```ts
import { configure } from "storybook/test";

configure({ asyncUtilTimeout: 3000 });
```

Which can go in `beforeAll` in your .storybook/preview file.
